lib/client.js

```diff
@@ -41,33 +42,47 @@ var Client = function Client(dsn, options) {
   // enabled if a dsn is set
   this._enabled = !!this.dsn;
 
+  var globalContext = this._globalContext = {};
+  if (options.tags) {
+    globalContext.tags = options.tags;
+  }
+  if (options.extra) {
+    globalContext.extra = options.extra;
+  }
 
   this.on('error', function(e) {}); // noop
 };
 node_util.inherits(Client, events.EventEmitter);
-var _ = Client.prototype;
+var proto = Client.prototype;
```

Comment:
Because it's synonymous with lodash/underscore. And nobody cares about saving 4 characters.